### PR TITLE
Feat/pipeline

### PR DESCRIPTION
**Pull request by ottonotto**
I added an eslint rule on this pr. should i not have? and instead, checkout to main - cut  branch - add rule - commit - and make another pr? I assume so, but this is a small lint change.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
           "arrowParens": "avoid"
         }
       ],
+      "eol-last":"error",
```
Comment (shams-ali) [inline .eslintrc.json:12]: eslint branch

Comment (ottonotto) [reply .eslintrc.json:12]: k

File: .github/workflows/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI/CD
+
+on:
+  push:
```
Comment (shams-ali) [inline .github/workflows/pipeline.yml:7]: push to main can never happen...no one should ever push directly to main

File: .github/workflows/pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      uses: actions/checkout@v2
+      
+    - name: Set up Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+        
+    - name: Install dependencies
+      run: npm install
```
Comment (shams-ali) [inline .github/workflows/pipeline.yml:33]: read up on npm ci

**Review by shams-ali — CHANGES_REQUESTED** [review]

File: .github/workflows/pipeline.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```
Comment (shams-ali) [inline .github/workflows/pipeline.yml:19]: this node version shuold generally be the node version your project is on...which shouold be the latest version of node

**Review by shams-ali — APPROVED** [review]

**Pull request merged**